src/view/filter.rs

```diff
@@ -2,26 +2,45 @@
 
 use crate::{Direction, LinkView, MultiView, NodeIndex, PortIndex, PortView};
 
-use context_iterators::{ContextIterator, FilterWithCtx, IntoContextIterator};
+use context_iterators::{ContextIterator, FilterWithCtx, IntoContextIterator, MapWithCtx};
 use delegate::delegate;
 
-/// Node filter used by [`NodeFiltered`].
+/// Node filter used by [`FilteredGraph`].
 pub type NodeFilter<Ctx> = fn(NodeIndex, &Ctx) -> bool;
+/// Port filter used by [`FilteredGraph`].
```

Comment:
Some clarification on the signature.
```suggestion
/// Link filter used by [`FilteredGraph`].
///
/// Ports that don't match this predicate will appear disconnected.
```